src/test/java/seedu/address/testutil/TypicalItems.java

```diff
@@ -14,11 +15,13 @@
 
     public static final Item APPLE = new ItemBuilder()
             .withId(1)
+            .withRecipe(new HashSet<>())
```

Comment:
Minor observation but item stores as a list, so why are we using hashset in itemBuilder?

Reply:
The constructor takes Set in at the moment, might look into this in future